dataland-document-manager/src/main/kotlin/org/dataland/documentmanager/services/InMemoryDocumentStore.kt

```diff
@@ -0,0 +1,37 @@
+package org.dataland.documentmanager.services
+
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Component
+
+/**
+ * A service that can be used to store binary blobs (documents) in memory
+ */
+@Component
+class InMemoryDocumentStore {
+    private val logger = LoggerFactory.getLogger(javaClass)
+    private val dataInMemoryStorage = mutableMapOf<String, ByteArray>()
+
+    /**
+     * Stores the provided dataset in memory identified by its sha256 hash.
+     */
+    fun storeDataInMemory(hash: String, data: ByteArray): String {
```

Comment:
In den anderen beiden Funktionen steht sha256hash hier nur hash. Das würde ich konsistent machen. Es ist ja überall das gleiche sollte also auch gleich heißen.

Reply:
In dem Service nennen wir den hash auch vermehrt documentId. Ich würde das hier wählen.